### ISSUE TITLE
ivar_variants_to_vcf.py collapses low quality ivar variant with high quality variant

### ISSUE THREAD

**Issue by Rohit-Satyam** [nf-core/viralrecon#337]
### Description of the bug

I noticed that ivar_variants_to_vcf.py is not behaving correctly. I was testing the difference between the phasing features of your `ivar_variants_to_vcf.py` and [phasing.py](https://github.com/TRON-Bioinformatics/covigator-ngs-pipeline/blob/master/bin/phasing.py), provided by COVIGATOR and it turns out that `ivar_variants_to_vcf.py` collapses bad quality variant with good quality variant
Eg:

```
MN908947.3	24129	A	G	30	29	36	1	1	37	0.0322581	31	0.508197	FALSE	CDS:ENSSASP00005000004	AAC	N	AGC	S
MN908947.3	24130	C	A	0	0	0	33	32	36	1	33	2.7703E-19	TRUE	CDS:ENSSASP00005000004	AAC	N	AAA	K
```
The A>G mutation at position 24129 is present in just 1. Therefore no matter that low quality variant call lie within same codon as high quality, they should be kept separate and only the high-quality calls must be collapsed. The current behaviour of your script is as follows:

```
MN908947.3	24129	.	AC	GA	.	ft	DP=31	GT:REF_DP:REF_RV:REF_QUAL:ALT_DP:ALT_RV:ALT_QUAL:ALT_FREQ	1:30:29:36:1:1:37:0.0322581	
MN908947.3	24256	.	T	C	.	ft	DP=33	GT:REF_DP:REF_RV:REF_QUAL:ALT_DP:ALT_RV:ALT_QUAL:ALT_FREQ	1:32:0:36:1:0:37:0.030303	
```

However it should be

```
MN908947.3	24129	.	A	G
MN908947.3	24130	.	C	A
``` 

Also, it is quite misleading to assign `AC>GA` with following tags of high quality variant `DP=31	GT:REF_DP:REF_RV:REF_QUAL:ALT_DP:ALT_RV:ALT_QUAL:ALT_FREQ	1:30:29:36:1:1:37:0.0322581
`

![image](https://user-images.githubusercontent.com/55479480/200149953-ef6a5f85-d08f-421b-8555-6c0d25b5f1a1.png)

**Comment by Rohit-Satyam** [comment]
@mattheww95 @saramonzon Any thoughts here?

**Comment by drpatelh** [comment]
@Rohit-Satyam presumably this issue is unrelated to https://github.com/nf-core/viralrecon/issues/321#issuecomment-1292610280 ? @saramonzon is this something we can fix in future releases?

**Comment by saramonzon** [comment]
Yes this is unrelated to the #321 fixes, we can totally fix it in the next releases, i will add it to the to do list :) Thanks for the tip!

**Comment by drpatelh** [comment]
Cool. Will move to the next milestone. Thanks @saramonzon !

**Comment by saramonzon** [comment]
Changes included here: https://github.com/nf-core/viralrecon/pull/469

Available in new version!
